view/adminhtml/templates/boltpay/js/replacejs.phtml

```diff
@@ -318,21 +317,19 @@ require([
      * Inject connect.js
      * return void
      */
-    var insertConnectScript = function() {
-        var scriptTag = document.getElementById('bolt-connect'),
-            publishableKey = getCheckoutKey();
+    var insertConnectScript = function(publishableKey) {
+        var scriptTag = document.getElementById('bolt-connect');
         if (scriptTag) {
-            scriptTag.setAttribute('data-publishable-key', publishableKey);
-            createOrder();
+            BoltCheckout.configure(cart, hints, callbacks);
             return;
         }
+        if (!publishableKey) return;
```

Comment:
let's do `console.error`

Reply:
@daisy1754 I will add it but I don't think it is necessary. Not having publishable key set is expected behavior. Please read the first two comments to Yevhen above.